src/main/java/org/takes/rq/RqForm.java

```diff
@@ -55,17 +55,25 @@
  * decorator before passing request to this class.
  *
  * <p>The class is immutable and thread-safe.
- *
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
- * @since 0.9
  * @link <a href="http://www.w3.org/TR/html401/interact/forms.html">Forms in HTML</a>
  * @see org.takes.rq.RqGreedy
+ * @since 0.9
  */
 @SuppressWarnings("PMD.TooManyMethods")
 public interface RqForm extends Request {
 
     /**
+     * The parameter separator.
+     */
+    String PARAM_SEPARATOR = "&";
```

Comment:
@mbarbieri I don't think our users should see these constants... they should be declared inside classes, as private. not here